pkg/eval/builtin_fn_misc.go

```diff
@@ -223,9 +247,9 @@ func init() {
 		"-source": source,
 
 		// Time
-		"esleep": sleep,
-		"time":   timeCmd,
-		"-time":  timeCmd,
+		"sleep": sleep,
```

Comment:
Don't remove `esleep`; make it an alias of `sleep` and deprecate it. We can remove it after 0.15.0.

Reply:
Done; although, I'm perplexed why it should be deprecated since anyone using it is not just using an experimental feature -- they're using an undocumented, halt-and-catch-fire, feature.